packages/block-editor/src/components/link-control/test/index.js

```diff
@@ -1324,7 +1331,8 @@ describe( 'Selecting links', () => {
 				} );
 
 				// Make sure focus is retained after submission.
-				expect( container ).toContainElement( document.activeElement );
+				// eslint-disable-next-line testing-library/no-node-access
+				expect( container.firstChild ).toHaveFocus();
```

Comment:
This is a bit more specific than checking if any element inside the rendered tree has the focus.